lib/resources/dstu2/immunization.yaml

```diff
@@ -0,0 +1,79 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization
+fields:
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: The vaccine that was administered or was to be administered.
+  example: |
+            {"vaccineCode": {
+              "coding": [
```

Comment:
The indenting should be 2 spaces from the "natural" alignment (not sure what to call that). If you were to align lines 12 and 13 like they were sibling nodes rather than parent/child nodes you would align them on the quotation marks, such that the v and c where in the same column. You should start your indentations from there, so in this case the c in "coding" should be under the first c in "vaccineCode".  I'm being picky since we are 1) trying to establish the consistent pattern and 2) these are on our public facing API docs which seem to be getting a lot of interest so we should be sure they look good/consistent

```
{"vaccineCode": {
   "coding": [
     {"system": "http://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=cvx",
      "code": "43"}],
   "text": "Engerix-B (obsolete)"}}
```

Note that the { is under the o in "coding".  This applies to several of your other examples as well.